x/zoneconcierge/keeper/keeper.go

```diff
@@ -0,0 +1,131 @@
+package keeper
+
+import (
+	"context"
+
+	corestoretypes "cosmossdk.io/core/store"
+	"cosmossdk.io/log"
+	storetypes "cosmossdk.io/store/types"
+	"github.com/babylonlabs-io/babylon/x/zoneconcierge/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	capabilitytypes "github.com/cosmos/ibc-go/modules/capability/types"
+	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
+	ibcexported "github.com/cosmos/ibc-go/v8/modules/core/exported"
+)
+
+type (
+	Keeper struct {
+		cdc              codec.BinaryCodec
+		storeService     corestoretypes.KVStoreService
+		ics4Wrapper      types.ICS4Wrapper
+		clientKeeper     types.ClientKeeper
+		connectionKeeper types.ConnectionKeeper
+		channelKeeper    types.ChannelKeeper
+		portKeeper       types.PortKeeper
+		authKeeper       types.AccountKeeper
```

Comment:
authKeeper, bankKeeper not used

Reply:
ditto, maybe keep them for now as by default modules depend on these two?